packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,40 @@
+// eslint-disable @typescript-eslint/naming-convention
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', () => {
+    beforeEach(() => {
+        cy.task('startEmu', { wipe: true, version: '2.4.3' });
+        cy.task('setupEmu', { mnemonic: 'all all all all all all all all all all all all' });
+
+        cy.task('startBridge');
+        cy.viewport(1080, 1440).resetDb();
+        cy.prefixedVisit('/');
+    });
+
+    afterEach(() => {
+        cy.task('stopEmu');
+    });
+
+    it(`Open Report form, fill bug report, submit report`, () => {
+        // goes to feedback
+        cy.getTestElement('@guide/button-open', { timeout: 20000 }).click();
+        cy.getTestElement('@guide/panel').should('exist');
+        cy.getTestElement('@guide/button-feedback').click();
+        cy.getTestElement('@guide/feedback/bug').click();
+
+        // gets first input in dropdown
+        cy.get('[class^=Select__Wrapper]').click();
+        cy.get('[id=react-select-2-option-0]').click();
```

Comment:
I don't like these selectors very much. They are not future proof. It may break upon simple lib update for example. It is better to assign own data-test attribute. I understand it might be complicated with react-select but we already have support data-test attributes in select component. Also you might cheat clicking here by using arrows on keyboard.

Reply:
I have created data-test attrand used them instead of selectors for this test.